gluten-core/src/main/scala/io/glutenproject/utils/Iterators.scala

```diff
@@ -110,8 +111,10 @@ private class PipelineTimeAccumulator[A](in: Iterator[A], pipelineTime: SQLMetri
   }
 
   private def tryFinish(): Unit = {
-    pipelineTime += accumulatedTime.getAndSet(
-      0L
+    pipelineTime += TimeUnit.NANOSECONDS.toMillis(
+      accumulatedTime.getAndSet(
```

Comment:
so this is due to `accumulatedTime` is changed during these days and it's nano-seconds(should use milliseconds)?